tests/Behat/Context/Ui/WishlistContext.php

```diff
@@ -159,6 +164,46 @@ public function iAddSelectedProductsToCart(): void
         $this->wishlistPage->addSelectedProductsToCart();
     }
 
+    /**
+     * @When /^the (product "([^"]+)") is stored in "(?P<filename>(?:[^"]|\\")*)"$/
+     */
+    public function productIsStoredInFile(ProductInterface $product, string $filename): void
+    {
+        /** @var ProductVariantInterface $productVariant */
+        $productVariant = $this->defaultVariantResolver->getVariant($product);
+
+        $data = [
+            $productVariant->getId(),
+            $product->getId(),
+            $productVariant->getCode()
+        ];
+
+        if ($this->getMinkParameter('files_path')) {
```

Comment:
if not + return: https://dev.to/tadeubdev/object-calisthenics-with-php-en-2h4p
Avoid unnecessary nesting, please.